# Joint positions in encoder units
int32[] positions